packages/client/test/sync/beaconsync.spec.ts

```diff
@@ -195,7 +197,7 @@ tape('[BeaconSynchronizer]', async (t) => {
   })
 
   t.test('should extend and set with a valid head', async (t) => {
-    const config = new Config({ transports: [] })
+    const config = new Config({ transports: [], cacheSize: 10000 })
```

Comment:
Not sure, an eventual more elegant way to solve this need of always passing in the - lowered - cache sizes in the tests might be to default the client to an ORDERED_MAP cache (and so get rid of this size problem) and only from the `bin/cli.ts` instantiation instantiate with LRU for cache_type.

Somewhat of a downside might be that tests are bit less "realistic" when we take another cache type for the tests.

No strong opinion here though, let me know if you have any thoughts or tendencies.

Reply:
Not sure if I have an opinion either way.  Doesn't the LRU cache type give significant performance improvements we would when running the client?

Reply:
Client should always be LRU - yes - this is just a question if the test setup would be a bit easier (and also somewhat less error prone, if one e.g. forgets to add the lower cache limits for config) if realized in the above-described way.

Anyhow, will leave for now, but still somewhat inclined to do, no super strong opinion though.